pmix/borrow.py

```diff
@@ -170,97 +240,30 @@ def merge_translation_file(merge: List[str], translation_dict: TranslationDict,
-    )
-    parser.add_argument(
-        '-M', '--merge_all', nargs='+',
-        help=('Merge into many files. To avoid ambiguity, this must be placed '
-              'after the source XLSForms.')
-    )
-    parser.add_argument(
-        '-C', '--correct', action='append',
-        help=('Mark a given file as correct. Text from these files will '
-              'disallow diverse translations from files not marked as '
-              'correct. This is a way to give files precedence for '
-              'translations.')
-    )
-    parser.add_argument(
-        '-D', '--no_diverse', action='store_true',
-        help='If text has diverse translations, do not borrow it.'
-    )
-    parser.add_argument(
-        '-d', '--diverse',
-        help=('Supply a language. Used without the --merge argument, '
-              'this creates a worksheet that shows only strings with '
-              'diverse translations for the supplied language.')
-    )
-    parser.add_argument(
-        '-a', '--add', action='append',
-        help=('Add a language to the resulting output. The translation file '
-              'will have a column for that language. Or, the merged XLSForm '
-              'will include columns for that language and have translations '
-              'for them if possible. This option can be supplied multiple '
-              'times.')
-    )
-    parser.add_argument(
-        '-i', '--ignore', action='append',
-        help=('A language to ignore when collecting and making '
-              'translations. This option can be supplied multiple times')
-    )
-    parser.add_argument(
-        '-c', '--carry', action='store_true',
-        help=('If translations are missing, carry over the same text from '
-              'the source language. The default is to leave missing.')
-    )
-    parser.add_argument(
-        '-o', '--outfile',
-        help=('Path to write output. If this argument is not supplied, then '
-              'defaults are used. If a command would produce multiple '
-              'outputs, then do not use this argument. Instead use '
-              '"--outdir".')
-    )
-    parser.add_argument(
-        '-O', '--outdir',
-        help=('A directory to use (and create if it does not exist) for '
-              'writing output. Ignored if -o is supplied. Defaults are used'
-              'for filenames. If neither outfile nor outdir are supplied, '
-              'then default filenames are used in the current directory.')
-    )
-
     args = parser.parse_args()
-    print(args)
-    borrow(
-        xlsxfiles=args.xlsxfile,
-        correct=args.correct,
-        merge=args.merge,
-        merge_all=args.merge_all,
-        no_diverse=args.no_diverse,
-        diverse=args.diverse,
-        add=args.add,
-        ignore=args.ignore,
-        carry=args.carry,
-        outfile=args.outfile,
-        outdir=args.outdir,
-    )
+    borrow_vars: List[str] = borrow.__code__.co_varnames
```

Comment:
Python has a better way: `vars(args)`. This gives you the parsed args as a dictionary. Would much prefer to go this route rather than what you have done.

Reply:
@jkpr Hah, yes I agree. I forgot about that.